mods/d2k/rules/structures.yaml

```diff
@@ -649,6 +649,7 @@ starport:
 	RevealsShroud:
 		Range: 3c768
 	RallyPoint:
+		Path: 3,3
```

Comment:
Didn't we remove default rally points from everything? Why add one for starport here? Even then, this should be 1,3.

Reply:
It is required for the units to not get stuck on top of each other.